drivers/ethernet/oa_tc6.c

```diff
@@ -0,0 +1,356 @@
+/*
```

Comment:
Please place the oa_tc6.c into a separate commit (could be introduced before the driver commit).